addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -26,9 +26,11 @@ var SlideDialog = Widget.extend({
      * @param {Object} el
      * @param {number} channel_id
      */
-    init: function (el, channelID) {
-        this._super(el, channelID);
-        this.channel_id = parseInt(channelID, 10);
+    init: function (el, data) {
+        this._super(el, data);
+        this.channel_id = parseInt(data.channelID, 10);
```

Comment:
maybe name it channelId using camel case ?

Reply:
Ok. I also updated string even it is it completely useless.